tests/client/main.cpp

```diff
@@ -4,12 +4,16 @@
 #define CATCH_CONFIG_MAIN
 
 #include "api_internal.h"
-#include "catch2\catch.hpp"
+#include "catch_wrapper.hpp"
 #include "ebpf_api.h"
 #pragma warning(push)
-#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
-#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
-                                // 'type2', possible loss of data
+#pragma warning(disable : 4100)  // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244)  // 'conversion' conversion from 'type1' to
+                                 // 'type2', possible loss of data
+#pragma warning(disable : 26451) // Arithmetic overflow
+#pragma warning(disable : 26450) // Arithmetic overflow
+#pragma warning(disable : 26439) // This kind of function may not throw. Declare it 'noexcept'
+#pragma warning(disable : 26495) // Always initialize a member variable
 #include "ebpf_verifier.hpp"
 #pragma warning(pop)
```

Comment:
I like your catch_wrapper solution.  I'd suggest the same for a verifier wrapper too.
That way if the verifier is fixed to avoid warnings, we only have to touch one place to remove the pragma.